frontend/src/pages/team/Pipelines/components/TeamPipeline.vue

```diff
@@ -4,16 +4,12 @@
             :to="{name: 'ApplicationPipelines', params: {id: pipeline.application.id}}"
             class="ff-pipeline-header flex gap-5 self-end items-center truncate"
         >
-            <span class="name">
-                {{ pipeline.name }}
-            </span>
-            <router-link
```

Comment:
are we sure we want to ditch the app link?

Reply:
Yep - definitely, it's really annoying, nobody clicks it intentionally, and clicking the pipeline header takes you to the Application > DevOps Pipeline view instead